src/controllers/events.js

```diff
@@ -85,12 +85,58 @@ export const postEvent = async (req, res, next) => {
   }      
 };
 
-export const getEvent = async (_req, res, next) => {
+export const getEvent = async (req, res, next) => {
+  try {
+    const id = req.params.id;
+    if (!mongoose.Types.ObjectId.isValid(id))
+      throw new ProblemError(
+        MESSAGE_TYPES.ERROR,
+        400,
```

Comment:
Maybe all error codes you can move in consts to be easier to change it if you want another error code for a specific error. Is just a suggestion is not wrong like this